user-guide/queries/query-url.md

```diff
@@ -0,0 +1,16 @@
+# Querying URLs
+
+Using a URL based data source requires that the URL return the [_results JSON format_](https://redash.io/help-onpremise/how-rd-works/data-source-results-format.md)
+
+The query itself inside Redash will simply contain the URL to be executed (i.e. http://myserver/path/myquery) and result with all the data it has - similar to `select * from db`.
```

Comment:
how about "... the data is has (the SQL equivalent would be `SELECT * FROM url`)."?